fastcrypto-proving/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "fastcrypto-proving"
+version = "0.1.0"
+license = "Apache-2.0"
+authors = ["Mysten Labs <build@mystenlabs.com>"]
+edition = "2021"
+publish = ["crates-io"]
+description = "zk-proof library used at Mysten Labs"
+repository = "https://github.com/MystenLabs/fastcrypto"
+
+[[bench]]
+name = "proving"
+harness = false
+
+[dependencies]
+
+[dev-dependencies]
+ark-bls12-381 = "0.3.0"
```

Comment:
I guess this is a temp (initial) backend for benchmarking purposes, until we end up with blst backend?

Reply:
It is, though I expect we will keep arkworks around for a while, for unit-testing purposes.